composer.json

```diff
@@ -11,24 +11,24 @@
         }
     ],
     "require": {
-        "php": "^5.6|^7.0",
-        "egeloen/json-builder": "^2.0|^3.0",
-        "symfony/dependency-injection": "^2.7|^3.0",
-        "symfony/form": "^2.7|^3.0",
-        "symfony/framework-bundle": "^2.7|^3.0"
+        "php": "^5.6 || ^7.0",
+        "egeloen/json-builder": "^2.0 || ^3.0",
+        "symfony/dependency-injection": "^2.0 || ^3.0 || ^4.0",
+        "symfony/form": "^2.7 || ^3.0 || ^4.0",
+        "symfony/framework-bundle": "^2.7 || ^3.0 || ^4.0"
```

Comment:
Could you restore the `|` way of writing alternative dependencies? No reason to change it, at least none I can tell.

Reply:
it just common in symfony ecosphere, no idea if it is a best practice.